### ISSUE TITLE
"at-rule-no-unknown" and mixins?

### ISSUE THREAD

**Issue by smlombardi** [stylelint/stylelint#1633]
All my mixins are flagged as warnings with `at-rule-no-unknown`.  This seems odd.

For example:
```scss
@mixin alert($bg, $bdr, $icon, $text: $pb-gray-700) {
  color: $text;
  background-color: $bg;
  background-image: url($icon);
  border-color: $bdr;
  i {
    color: $bdr;
  }
}
```
and
```scss
.alert-danger {
  @include alert($state-danger-bg, $state-danger-border, $icon-danger);
}
```
Which are both valid scss are flagged as warnings.  I had to disable the rule.

**Comment by jeddy3** [comment]
Dupe of https://github.com/stylelint/stylelint/issues/1612

> I had to disable the rule.

It's fine to do that. Not [all rules are applicable](https://github.com/stylelint/stylelint/blob/master/docs/developer-guide/rules.md#let-everyone-know) to non-standard syntax.

**Comment by Nirazul** [comment]
@jeddy3 
The documentation on the rule is quite sparse. Maybe it's a good idea to explain what is considered "`unknown`"

**Comment by jeddy3** [comment]
>  Maybe it's a good idea to explain what is considered "unknown".

Done in #1612 and https://github.com/stylelint/stylelint/pull/1613 - the website will update with the next release.

**Comment by Nirazul** [comment]
Always a step ahead 😄
